dask/dataframe/csv.py

```diff
@@ -40,7 +40,15 @@ def bytes_read_csv(b, header, kwargs, dtypes=None, columns=None,
         bio.write(header)
     bio.write(b)
     bio.seek(0)
-    df = pd.read_csv(bio, **kwargs)
+
+    try:
+        df = pd.read_csv(bio, **kwargs)
+    except (csv.Error, pd.io.common.CParserError) as e:
+        raise ValueError(
+            "There was an error in parsing a CSV block. This may be due to "
+            "the line terminator appearing in a quoted value. Original error: "
+            "%s" % str(e))
```

Comment:
Parse errors occur for a variety of reasons.  I suspect that line-terminators are not the common case so I'm a bit hesitant to call it out by default.  Is there something we can check in the text of the pandas error to see if we should explain things?